src/Connection/MessageReceivers/HandDataHandler.ts

```diff
@@ -32,12 +32,14 @@ export class HandDataHandler {
     checkForHandData = () => {
         const handFrame = this.latestHandDataItem;
 
-        if (handFrame) {
-            this.latestHandDataItem = undefined;
-            // Wrapping the function in a timeout of 0 seconds allows the dispatch to be asynchronous
-            setTimeout(() => {
-                HandDataManager.handleHandFrame(handFrame);
-            });
+        if (!handFrame) {
```

Comment:
🤷 Personally like simple if statements like this to be one lined 
`if(!handFrame) return;`

but is preference so if you prefer {} then just resolve this comment.

Reply:
Updated